lineman/app/components/dashboard_page/dashboard_page_controller.coffee

```diff
@@ -34,14 +34,14 @@ angular.module('loomioApp').controller 'DashboardPageController', ($rootScope, $
     @currentBaseQuery = ThreadQueryService.filterQuery(@filter)
     if @displayByGroup()
       _.each @groups(), (group) =>
-        @views.groups[group.key] = ThreadQueryService.groupQuery(group, { filter: @filter, queryType: 'all' })
+        @views.groups[group.key] = ThreadQueryService.groupQuery(group, { filter: 'show_muted', queryType: 'all' })
```

Comment:
'display by group' != 'muted' in my mind, it just happens to be that way in this iteration. Why doesn't `@filter` work here?